snekbox/api/resources/eval.py

```diff
@@ -51,6 +57,7 @@ def on_post(self, req: falcon.Request, resp: falcon.Response) -> None:
 
         >>> {
         ...     "input": "print(1 + 1)"
+        ...     "args": ["-m", "timeit"] # This is optional
```

Comment:
There needs to be a comma between the kv pairs.

Reply:
The response no longer matches the request.

Reply:
Add a brief description of the `args` parameter to the docstring. I'm concerned that merely an example would not be sufficiently clear.